doc/nrf/libraries/modem/sms.rst

```diff
@@ -23,14 +23,23 @@ In addition, AT commands are also used to send SMS messages.
 SMS notifications are received using AT commands, but those are not visible for the users of this module.
 The module automatically acknowledges the SMS messages received on behalf of each listener.
 
+The SMS module reads SMS service center number using ``AT+CSCA?`` AT command.
+Most applications do not need to set custom SMS service center number but default value is enough.
+However, if your application requires custom SMS service center number, you can set it using ``AT+CSCA`` AT command.
+Because ``AT+CSCA`` AT command is not supported by the modem firmware, you can use the :ref:`lib_gcf_sms_readme` library.
```

Comment:
```suggestion
The SMS module reads the SMS service center number using the ``AT+CSCA?`` AT command.
Most applications do not need to set a custom SMS service center number because the default value is enough.
However, if your application requires a custom SMS service center number, you can set it using the ``AT+CSCA`` AT command.
Because ``AT+CSCA`` AT command is not supported by the modem firmware, you can use the :ref:`lib_gcf_sms_readme` library.
```